Indexing/ClauseCodeTree.cpp

```diff
@@ -731,22 +720,20 @@ void ClauseCodeTree::ClauseMatcher::enterLiteral(CodeOp* entry, bool seekOnlySuc
     linfoCnt/=2;
   }
 
-  LiteralMatcher* lm;
-  Recycler::get(lm);
+  Recycled<LiteralMatcher, NoReset> lm;
   lm->init(tree, entry, lInfos.array(), linfoCnt, seekOnlySuccess);
-  lms.push(lm);
```

Comment:
This pattern I understand!

(also, it's reassuring that `lm` goes out of scope immediately after the move)